index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href= "styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
```

Comment:
[  OPTIONAL ]
I suggest you use an h1 tag to wrap the text " hello Microverse " as using the right semantic tags helps the browser read the content properly.